java/client/src/main/java/glide/managers/CommandManager.java

```diff
@@ -34,21 +34,52 @@ public class CommandManager {
     /**
      * Build a command and send.
      *
-     * @param command The command to execute
+     * @param requestType Redis command type
+     * @param arguments Redis command arguments
      * @param responseHandler The handler for the response object
      * @return A result promise of type T
      */
     public <T> CompletableFuture<T> submitNewCommand(
-            Command command, RedisExceptionCheckedFunction<Response, T> responseHandler) {
+            RedisRequestOuterClass.RequestType requestType,
```

Comment:
```suggestion
            RequestType requestType,
```

Reply:
We already have RequestType in scope of the package.  So we cannot import the protobuf RequestType directly.

Reply:
We removed our `RequestType` enum AFAIK
`RequestType` from protobuf is already imported in this file, just use it